### PR TITLE
Attempt No.1

### PR DESCRIPTION
**Pull request by TurnbulA**

### CODE REVIEW

File: app/scss/addProject.scss [hunk 1]
```diff
@@ -25,9 +25,9 @@
     display: none;
     transition: all 200ms ease-in-out;
     border-radius: 2rem;
-    &--active {
```
Comment (mali891) [inline app/scss/addProject.scss]: This one was correct

File: app/scss/searchBar.scss [hunk 2]
```diff
@@ -35,20 +35,21 @@
...
     }
   }
+  &__text {
+    color: $blackSqueeze;
+    font-size: 3.5rem;
+    margin-bottom: 0.5rem;
+  }
+  &__reset-button {
+    border: none;
+    color: #f64a4a;
```
Comment (mali891) [inline app/scss/searchBar.scss]: Can this be a variable

File: app/scss/searchBar.scss [hunk 3]
```diff
@@ -55,3 +49,11 @@
     }
   }
 }
+
+.c-reset-button {
+  border: none;
+  color: #f64a4a;
```
Comment (mali891) [inline app/scss/searchBar.scss]: Variable?

Comment (TurnbulA) [reply app/scss/searchBar.scss]: Yes?

File: app/scss/searchBar.scss [hunk 4]
```diff
@@ -52,7 +52,7 @@
 
 .c-reset-button {
   border: none;
-  color: #f64a4a;
+  color: $usedRed;
```
Comment (mali891) [inline app/scss/searchBar.scss]: yes familyyyyy

**Pull request merged**